plugins/modules/ipauser.py

```diff
@@ -483,32 +484,26 @@
 def find_user(module, name, preserved=False):
```

Comment:
As `user-show` is returning the preserved setting for all users, what do you think about removing the `preserved` parameter and using `res_find.get("preserved")` later on where `res_find_preserved` is used? This would remove an additional call for preserved users or if a user can not be found.

Reply:
I agree. I tried to make less changes to the code, specially function signatures.

Reply:
Fixed. This has reduced one call to IPA API.